napari/_qt/qt_viewer.py

```diff
@@ -198,7 +198,7 @@ def __init__(self, viewer, welcome=True):
         )
         self.canvas.events.resize.connect(self.scale_bar._on_position_change)
 
-        self._show_welcome = welcome
+        self._show_welcome = welcome and config.allow_welcome_visual
```

Comment:
In this case, it wasn't the crashes. It was because I wanted to set breakpoints in image visual code, but I didn't want them to hit until my octree images were created. Incredibly minor, yet a developer should have the ability to toggle something like that on or off for a number of reasons.

I wonder if you can push through the worse possible version of #1183 and then learn from that and improve it. Some type of config file is pretty key. Command line args and env vars are not bad, but a config files is arbitrarily big, structured, can be backed up and have multiple versions, etc.